test/test_bort.F90

```diff
@@ -6,9 +6,12 @@
 ! Ed Hartnett 3/12/23
 program test_bort
   implicit none
+  character*2 char_short
+  character*30 char_30
 
   integer :: num_args, len, status
   character(len=32) :: sub_name, test_case
+  CHARACTER ADN30
   
```

Comment:
This should be declared as `character*6 adn30` or `character(len=6) adn30`, because it will return up to a 6-byte character string.

Reply:
Fixed.